src/libmongoc/tests/test-mongoc-dns.c

```diff
@@ -74,6 +74,36 @@ typedef struct {
 } context_t;
 
 
+/*
+ *--------------------------------------------------------------------------
+ *
+ * _mongoc_host_list_push --
+ *
+ *       Add a host to the front of the list and return it.
+ *
+ * Side effects:
+ *       None.
+ *
+ *--------------------------------------------------------------------------
+ */
+mongoc_host_list_t *
```

Comment:
```suggestion
static mongoc_host_list_t *
```

Suggest defining function as static since it is not accessed outside of this file.